lib/reek/smells/smell_detector.rb

```diff
@@ -55,6 +55,12 @@ def exception?(context)
         context.matches?(value(EXCLUDE_KEY, context, DEFAULT_EXCLUDE_SET))
       end
 
+      def self.todo_configuration_for(smells)
+        default_exclusions = default_config['exclude']
+        exclusions = default_exclusions + smells.map(&:context)
+        { smell_type => { 'exclude' => exclusions.uniq } }
```

Comment:
Nice and concise.
The only thing that remains is too fix the UsesClassMethod smell, but that is something bigger for another pull request ;)